index.html

```diff
@@ -1724,6 +1724,78 @@ <h4>CredentialRegistry</h4>
         </pre>
       </section>
 
+      <section>
+        <h4>OpenID4 Verifiable Credential Issuance</h4>
+        <p>The <code>OID4VCI</code> service allows publication of the <a href="https://openid.net/openid4vc/">OpenID for Verifiable Credential Issuance (OID4VCI)</a> domain, where the <code>.well-known</code> files (<code>oauth-authorization-server</code> & <code>openid-credential-issuer</code>) can be found. These files MUST contain the endpoints and meta data for <a href="https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html">OpenID for Verifiable Credential Issuance (OpenID4VCI)</a>.</p>
+        <table class="simple" style="width: 100%;">
```

Comment:
how about ...

```suggestion
The service endpoint `id` MUST be the Credential Issuer Identifier to get the Credential Issuer Metadata as described in Section [Credential Issuer Metadata](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#credential-issuer-metadata) of [OpenIDVCI](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html).

The service endpoint MAY contain additional Credential Issuer Metadata parameters as per [Credential Issuer Metadata](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#credential-issuer-metadata).
```

Reply:
Please ignore the Object / metadata for now, I'd recommend only the issuer identifier for the `id` and update to object in future as needed. For now I don't really see the need and it might actually lead to confusion.

Reply:
```suggestion
        <table class="simple" style="width: 100%;">       
<p>The service endpoint `id` MUST be the Credential Issuer Identifier to get the Credential Issuer Metadata as described in Section [Credential Issuer Metadata](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#credential-issuer-metadata) of [OpenIDVCI](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html).</p>
```

Reply:
Integrated in https://github.com/w3c/did-spec-registries/pull/513/commits/e7c2827d78b79bbb4af97036673d4df18b116518